appveyor-ubuntu.yml

```diff
@@ -184,12 +189,82 @@ for:
       - "pip install -e \".[pre-dev]\""
 
       - "pip install -e \".[dev]\""
-      - sh: "pytest -vv tests/integration/buildcmd --json-report --json-report-file=TEST_REPORT-integration-buildcmd.json"
+      - sh: "pytest -vv tests/integration/buildcmd --ignore=tests/integration/buildcmd/test_build_terraform_applications.py --ignore=tests/integration/buildcmd/test_build_terraform_applications_other_cases.py --json-report --json-report-file=TEST_REPORT-integration-buildcmd.json"
 
       # Set JAVA_HOME to java11
       - sh: "JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64"
       - sh: "pytest -vv tests/integration/buildcmd/test_build_cmd.py -k test_building_java11_in_process --json-report --json-report-file=TEST_REPORT-integration-buildcmd-java11"
 
+  # Local ZIP  Terraform Build integ testing
+  -
+    matrix:
+      only:
+        - image: Ubuntu
+          configuration: LocalZipTerraformBuildIntegTesting
+
+    test_script:
+      # Pre-dev Tests
+      - "pip install -e \".[pre-dev]\""
+
+      - "pip install -e \".[dev]\""
+      - sh: "pytest -vv -n 4 tests/integration/buildcmd/test_build_terraform_applications.py::TestBuildTerraformApplicationsWithZipBasedLambdaFunctionAndLocalBackend_0 --json-report --json-report-file=TEST_REPORT-integration-buildcmd.json"
```

Comment:
I like the `-n 4` there is also an auto variable you could use on this. Can this level of parallelization cause flaky tests?